components/expenses/PayoutBankInformationForm.js

```diff
@@ -72,8 +72,109 @@ const requiredFieldsQuery = gqlV2`
   }
 `;
 
-const RequiredFields = ({ disabled, getFieldName, formik, host, currency }) => {
-  const { loading, error, data } = useQuery(requiredFieldsQuery, {
+const Input = props => {
+  const { input, getFieldName, disabled, currency, loading, refetch, formik, host } = props;
+  const fieldName =
+    input.key === 'accountHolderName' ? getFieldName(`data.${input.key}`) : getFieldName(`data.details.${input.key}`);
+
+  if (input.type === 'text') {
+    const validate = input.validationRegexp
+      ? value => (new RegExp(input.validationRegexp).test(value) ? undefined : `Invalid ${input.name}`)
+      : undefined;
+    return (
+      <Box key={input.key} mt={2} flex="1">
+        <FastField name={fieldName} validate={validate}>
+          {({ field, meta }) => (
+            <StyledInputField label={input.name} required error={meta.touched && meta.error}>
```

Comment:
This `meta.touched && meta.error` creates a strange behavior when you first try to submit the form without touching anything: the "Review expense summary" gets disabled but we don't display the error. I think we can ignore the `meta.touched`.

Reply:
We actually need this, otherwise the fields will be rendered displaying error messages like "Invalid Account Number" from the start.